packages/flash-calendar/src/components/Calendar.tsx

```diff
@@ -68,8 +68,25 @@ export interface CalendarProps extends UseCalendarParams {
   calendarMonthHeaderHeight?: number;
   /** Theme to customize the calendar component. */
   theme?: CalendarTheme;
+  /** Enable dark mode */
+  darkMode?: boolean;
 }
 
+const calendarContext = createContext<{ darkMode?: boolean } | undefined>(
```

Comment:
instead of `darkMode`, let's use the idea from above: 
```
type CalendarContextType = {
  /** The overridden color scheme */
  colorScheme?: ColorSchemeName
}

const calendarThemeContext = createContext<CalendarContextType | undefined>(undefined)